spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe AdminUser do
+  it 'return type should be User' do
+    testuser = AdminUser.find_or_create
+    expect(testuser).to be_a_kind_of(User)
+  end
+  context 'no existing admin' do
```

Comment:
For context blocks, the text should introduce a condition with "when"/"where" and use complete phrases. For example, `'when there is no existing admin'`.